src/collection-view.js

```diff
@@ -279,19 +268,55 @@ const CollectionView = Backbone.View.extend({
       this._showEmptyView();
     } else {
       this.triggerMethod('before:render:children', this);
-      this._startBuffering();
       this._showCollection(models);
-      this._endBuffering();
       this.triggerMethod('render:children', this);
     }
   },
 
+  _createView(model, index) {
+    const ChildView = this._getChildView(model);
+    const childViewOptions = this._getChildViewOptions(model, index);
+    const view = this.buildChildView(model, ChildView, childViewOptions);
+    view._parent = this;
+
+    monitorViewEvents(view);
+
+    // set up the child view event forwarding
+    this._proxyChildEvents(view);
```

Comment:
just for funzies, what is the perf test difference if you comment out this line?